.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
@avinashsharma080 please remove this file

Reply:
Sure